src/lib/util.js

```diff
@@ -13,23 +13,24 @@ let cookieTime = (days) => {
 
```

Comment:
to avoid having transpile bugs with JSX and renderIf you can implament it using the following

``` javascript
export const renderIf = (test, component) => {
   if (test) return comoponent
}
```